packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts

```diff
@@ -106,7 +106,11 @@ export class CheqdAnonCredsRegistry implements AnonCredsRegistry {
           schemaId: `${schema.issuerId}/resources/${schemaResource.id}`,
         },
         registrationMetadata: {},
-        schemaMetadata: {},
+        schemaMetadata: {
```

Comment:
We don't support cheqd with indy sdk, so this isn't needed i guess?

Reply:
Anoncreds api is throwing an error if I don't include it
 https://github.com/hyperledger/aries-framework-javascript/blob/main/packages/anoncreds/src/AnonCredsApi.ts#L264

Reply:
What is the error it throws? Are you using indy sdk?

Reply:
Yea I tried using anoncreds-rs module for anoncreds and indysdk for wallet. It panics at that line and returns invalid did:indy did even though I'm using did:cheqd

Reply:
In that case it's probably an issue with using AnonCreds RS and Indy SDK together. It's not intended to be used like that, and thus I don't think we should add code to support the flow

Reply:
Removed that change